### ISSUE TITLE
U2+ 3.10a 1581 emulation not working in C128

### ISSUE THREAD

**Issue by mhansel7399** [GideonZ/1541ultimate#255]
I just updated my U2+ to 3.10a this evening. When I mount a D81 image in C128 and go do a DIRECTORY (or LOAD"$",8 and LIST), I get nothing. If I reboot into C64 mode, I can view the file listing on the disk. The D81 image is for the C128 (DESTERM128.D81). Am I missing something in my settings? Please advise.

**Comment by xlar54** [comment]
When you say you get "nothing", does it freeze up, give you just a Ready
prompt? Can you define "nothing" a little better?

**Comment by mhansel7399** [comment]
It gives me a Ready prompt. And when I try to load something it sits at "Searching for {filename}"

**Comment by markusC64** [comment]
It *might* be a hardware problem with your C128 or with the cable connecting your U2+.

You see, in C128 mode, fast serial mode is used for the C128, which requires an extra line in the cable (it should be there, but it might be damaged). Or the CIA does not generate the SRQ signal (that's the name of the extra line).
In C64 mode, the SRQ line is not used at all, so it can behave quite different.

**Comment by mhansel7399** [comment]
Ok, so I replaced the cable between the C128 and the U2+ and am experiencing the same results. This may not be the forum for this, but can someone help me troubleshoot if the CIA is not doing what it is supposed to? Sorry, but my electronics diagnostics stop at looking at OBVIOUS things. Thanks in advance!
--Matt

**Comment by markusC64** [comment]
By the way: Is your C128 PAL or NTSC?

Dunno if this makes a difference, but NTSC might have been tested less. In case of small timing, that might make a difference. Anyway, I don't believe that NTSC might be a reson, but to be sure...

Anyway, this video shows how to test: https://www.youtube.com/watch?v=P5_oew0m7Z8 - but you'll need a Check64...

**Comment by mhansel7399** [comment]
It is an NTSC unit. I did add the VDC RAM upgrade a while back (at least 2 years ago) - don't know if that will make a difference. I will watch the video and see what I can learn from that. 
So, I learned that the Check64 is a rarity right now, and to do real diagnostics, I need one. =( I wonder if Ray Carlsen can check it out?

**Comment by mhansel7399** [comment]
So color me "not so smart". Last night I spent some more time troubleshooting. I won't go into the gory details, but I removed the WiModem OLED unit, and the C128 and 1581 emulation worked fine. So, a little "Googling" later, I was reminded of the UP9600 hack. I followed the instructions to remove that hack from the WiModem, and plugged it back in, and the C128 and 1581 emulation work fine! I appreciate all of the support and suggestions. Thanks for helping me figure it out.
